README.md

```diff
@@ -326,7 +325,7 @@ TODO: code and documentation need to be added.
 
 # License
 
-Copyright 2019 Square Inc.
+Copyright 2020 Square Inc.
```

Comment:
IANAL but if this code was written in 2019, I believe the copyright header can stay like this.

Reply:
correct, copyrights should only change for new files

Reply:
TIL!
Reverted back